docs/docs/recipes/manage-users/using-admin-console.md

```diff
@@ -23,36 +25,56 @@ Support the partial search on the user properties: name, username, email, or pho
 
 Admin users could create a new account through Admin Console in the name of an end-user.
 
-1. On the "User Management" panel, click the "+ Add User" button.
-2. In the dialog, fill out the username and full name, and click the "Add User" button to confirm.
-3. After successfully adding, click the eye icon to show or hide the actual password.
-4. Copy the **username** and **initial password** to send to the end-user.
+On the "User Management" panel, click the "+ Add User" button.
+In the dialog, fill out the username and full name, and click the "Add User" button to confirm.
+
+After successfully adding, copy the **username** and **initial password** to send to the end-user.
 
 :::caution
 After closing the "Add User" dialog, you can no longer view the password.
 If you forget to keep it, you have to [reset the password](#reset-user-password).
 :::
 
-## View user profile
+## View and update user profile
+
+On the "User Management" panel, click the user table row to jump to the "**User details**" panel.
```

Comment:
ditto. you can upload screenshots now

Reply:
plan to add in another PR. maybe keep it for now?

Reply:
wait for Guamian to provide the screenshot, and will add them later.